src/main/java/com/redhat/cloud/policies/app/rest/AdminService.java

```diff
@@ -169,6 +177,10 @@ public Response syncToEngine(@QueryParam("token") String token) {
     @Path("/verify")
     public Response findOrphans() {
 
+        if (lightweightEngineConfig.isEnabled()) {
+            return Response.status(503).entity("Verify unavailable when the lightweight engine is enabled").build();
```

Comment:
a very simple unit test would be great so we have something like an insurance when we change something somewhere in the code.